index.js

```diff
@@ -203,7 +207,7 @@ app.post('/login', (req, res) => {
 
 app.get('/login', (req, res) => {
     try {
-        const user = checkIfLoggedIn(req.body.userLoginSession);
+        const user = isUserLoggedIn(req.body.userLoginSession);
```

Comment:
`isUserLoggedIn` name suggests that the function will return a boolean value. But you're assigning this value to `user`, expecting it to be a user object. This doesn't look right. Can you fix this please?

Reply:
Found out that it uses function findLoggedInUser, that returns user, and this function return same user. so I've just deleted it.